src/app.js

```diff
@@ -320,6 +347,8 @@ var run = function () {
 
     currentFile = file
 
+    files.event.trigger('fileFocus', [file])
```

Comment:
Also, do we need `editorSyncFile` above, won't it be called from the event procedure later?